src/main/java/EtfRecommendService/etf/EtfRestController.java

```diff
@@ -27,37 +27,47 @@ public ResponseEntity<EtfResponse> read(@RequestParam(defaultValue = "1") int pa
                                             @RequestParam(required = false, defaultValue = "") String keyword,
                                             @RequestParam(defaultValue = "weekly") String period) {
         Pageable pageable = PageRequest.of(page - 1, size);
-        EtfResponse etfResponse = etfService.readAll(pageable, theme,keyword, period);
+        EtfResponse etfResponse = etfService.readAll(pageable, theme, keyword, period);
         return ResponseEntity.status(HttpStatus.OK).body(etfResponse);
     }
 
+    //페이징 없는 전체조회
+    @GetMapping("/all")
+    public ResponseEntity<EtfAllResponse> readAll(
```

Comment:
`/search`가 더 나을 것 같네요
```suggestion
    //페이징 없는 전체조회
    @GetMapping("/search")
    public ResponseEntity<EtfAllResponse> readAll(
```